### PR TITLE
Initialize Nuxt

### PR DESCRIPTION
**Pull request by TeddTech**
Scaffolding for nuxt poc

### CODE REVIEW

**Comment by greatislander** [comment]
For review:

- @jobara
- @michelled

File: we_count/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```
Comment (jobara) [inline we_count/.editorconfig]: We typically use 4 spaces for our projects. Is there a specific reason to use 2 spaces for this work?

File: we_count/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  env: {
+    browser: true,
+    node: true
+  },
+  parserOptions: {
+    parser: 'babel-eslint'
+  },
+  extends: [
+    '@nuxtjs',
+    'plugin:nuxt/recommended'
```
Comment (jobara) [inline we_count/.eslintrc.js]: We have a common [eslint config](https://github.com/fluid-project/eslint-config-fluid) for our javascript projects. What are the main differences and is there a reason to be using the one supplied by nuxt over the fluid version?

File: we_count/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# we_count
+
+> We Count website
+
+## Build Setup
+
+``` bash
+# install dependencies
+$ npm run install
```
Comment (jobara) [inline we_count/README.md]: For the standard npm scripts you don't need to call `run` before the script name. https://docs.npmjs.com/misc/scripts

File: we_count/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+# serve with hot reload at localhost:3000
+$ npm run dev
+
+# build for production and launch server
+$ npm run build
+$ npm run start
```
Comment (jobara) [inline we_count/README.md]: what is the difference between the `build` and `start` script. It would be good to articulate what to use which.

**Comment by jobara** [comment]
@TeddTech we typically create an issue first and then prefix the commits and the PR with the Issue number. This helps keep track of what the work was for. It would probably be good practice for future work to follow a similar pattern.

**Comment by TeddTech** [comment]
> @TeddTech we typically create an issue first and then prefix the commits and the PR with the Issue number. This helps keep track of what the work was for. It would probably be good practice for future work to follow a similar pattern.

File: we_count/README.md (hunk 4 above)
Comment (TeddTech) [reply we_count/README.md]: There same unless you specify specific task to complete during build. In that case `npm run build` will also complete these tasks when the script runs. I have just been using `npm run dev` so far

File: we_count/.eslintrc.js (hunk 2 above)
Comment (TeddTech) [reply we_count/.eslintrc.js]: Not at all. It was the default linter configuration. Will have it changed for the next pull request.

File: Pipfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
[the remark this answers is not in the document]
Comment (TeddTech) [reply Pipfile]: I use pipenv to set up my virtual environment.

File: we_count/.editorconfig (hunk 1 above)
Comment (TeddTech) [reply we_count/.editorconfig]: No reason just default. Will change for next pr

**Pull request closed without merging**